src/modules/material/material.service.js

```diff
@@ -0,0 +1,31 @@
+const Material = require('./material.model');
```

Comment:
Please, move all newly added logic for materials to another merge request to merge it in development as quickly as possible.